<launch>
  <node pkg="pcl_csf" type="pcl_csf_node" name="pcl_csf_node" output="screen">
    <param name="bSloopSmooth"     type="bool"   value="true" />
    <param name="cloth_resolution" type="double" value="0.5" />
    <param name="rigidness"        type="int"    value="3" />
    <param name="time_step"        type="double" value="0.65" />
    <param name="class_threshold"  type="double" value="0.01" />
    <param name="interations"       type="int"    value="500" />

    <param name="pcd_path"         type="string" value="/home/yhs/Downloads/LOAM/GlobalMap.pcd" />
    <param name="resolution"       type="double" value="0.05" />
    <param name="z_min"            type="double" value="-2.0" />
    <param name="z_max"            type="double" value="5.0" />
    <param name="MeanK"            type="int"    value="60" />
    <param name="StddevMulThresh"  type="double" value="0.1" />
  </node>
</launch>